.coveragerc

```diff
@@ -0,0 +1,4 @@
+[report]
+exclude_lines =
+    pragma: no cover
+    if TYPE_CHECKING:
```

Comment:
@feluelle  could you add a comment to the open issue that explains this is not working

Reply:
I have removed it as it does not work.